react-components/src/components/Architecture/SettingsButton.tsx

```diff
@@ -266,74 +281,78 @@ function createSlider(command: BaseSliderCommand, t: TranslateDelegate): ReactNo
   );
 }
 
-function createDropdownButton(command: BaseOptionCommand): ReactNode {
+function DropdownButtonComponent({ command }: { command: BaseOptionCommand }): ReactNode {
   // @update-ui-component-pattern
   const [isVisible, setVisible] = useState(true);
-  const [uniqueId, setUniqueId] = useState(0);
 
   useOnUpdate(command, () => {
     setVisible(command.isVisible);
-    setUniqueId(command.uniqueId);
   });
   // @end
 
   if (!isVisible) {
     return null;
   }
 
-  return (
-    <DropdownButton
-      key={uniqueId}
-      inputCommand={command}
-      placement={'bottom'}
-      usedInSettings={true}
-    />
-  );
+  return <DropdownButton inputCommand={command} placement={'bottom'} usedInSettings={true} />;
 }
 
-function createFilterButton(command: BaseFilterCommand): ReactNode {
+function FilterButtonComponent({ command }: { command: BaseFilterCommand }): ReactNode {
   command.initializeChildrenIfNeeded();
 
   // @update-ui-component-pattern
   const [isVisible, setVisible] = useState(true);
-  const [uniqueId, setUniqueId] = useState(0);
 
   useOnUpdate(command, () => {
     setVisible(command.isVisible);
-    setUniqueId(command.uniqueId);
   });
   // @end
 
   if (!isVisible) {
     return null;
   }
-  return (
-    <FilterButton
-      key={uniqueId}
-      inputCommand={command}
-      placement={'bottom'}
-      usedInSettings={true}
-    />
-  );
+  return <FilterButton inputCommand={command} placement={'bottom'} usedInSettings={true} />;
 }
 
 const SliderDiv = styled.div`
+  width: 100%;
   display: flex;
   flex-direction: column;
   align-items: start;
   justify-content: space-between;
-  gap: 8px;
-  padding: 4px 8px;
+  gap: 4px;
   font-size: 14px;
 `;
 
 const StyledSlider = styled(Slider)`
   offset-anchor: right top;
   float: center;
   display: flex;
+
   justify-content: space-around;
 `;
 
+const StyledSectionHeader = styled.span`
+  color: var(--text-icon-muted, rgba(0, 0, 0, 0.55));
+  font-feature-settings: 'cv05' on;
+
+  /* Label/XSmall */
+  font-family: Inter;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px; /* 166.667% */
+  letter-spacing: -0.036px;
+`;
+
+const StyledToggleContainer = styled(Flex).attrs({
+  direction: 'row'
+})`
+  width: 100%;
+  justify-content: start;
+  gap: 8px;
+`;
```

Comment:
```suggestion
const StyledToggleContainer = styled.div`
  display: flex;
  width: 100%;
  justify-content: start;
  gap: 8px;
`;
```
`row` is default value for the `flex-direction` and it's not inherited as far as I know. We can simply define this as a `div`. It's a bit of a gray area, but if we're gonna override styles for cogs' `Flex` component, we can might as well define it as a `div`